src/Core/Upgrade/UpgradeApiResourceVisitor.php

```diff
@@ -443,6 +443,9 @@ private function legacyOperationsToOperations($legacyOperations, bool $isCollect
 
             $method = $operation['method'] ?? strtoupper($operationName);
             unset($operation['method']);
+            if(!in_array($operationName, ['get', 'post', 'put', 'patch', 'delete'])) {
+                $operation['name']= $operationName;
```

Comment:
I'll keep this patch, it's a good idea thanks!

Reply:
@soyuka Great, I am very pleased to have been helpful.